oci/jenkins.py

```diff
@@ -61,6 +61,9 @@ def system_tests(self, custom_repos=None, engine_version="master",
             suite_type (str): Suite type to run. For available suite types see
                 SUITE_TYPE popup menu at:
                 https://jenkins.ovirt.org/job/ovirt-system-tests_manual/build
+            coverage (boolean): Report tests coverage. For available coverage options
+                see COVERAGE popup menu at:
+                https://jenkins.ovirt.org/job/ovirt-system-tests_manual/build
```

Comment:
Since this is a bool, there is no need to go to gerrit and look at the popup menu.

Reply:
fixed.